src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -318,27 +329,26 @@ class InviteMembersModal extends AsyncComponent<Props, State> {
         <Heading>
           <IconMail size="lg" />
           {t('Invite New Members')}
-          {!this.willInvite && (
-            <QuestionTooltip
-              title={t(
-                `You do not have permission to directly invite members. Email
-                 addresses entered here will be forwarded to organization
-                 managers and owners; they will be prompted to approve the
-                 invitation.`
-              )}
-              size="sm"
-              position="bottom"
-            />
-          )}
         </Heading>
         <Subtext>
           {this.willInvite
             ? t('Invite new members by email to join your organization.')
             : t(
                 `You don’t have permission to directly invite users, but we’ll
-                 send a request on your behalf.`
+                 send your request to organization managers and owners for approval.`
               )}
         </Subtext>
+        {!this.willInvite && (
+          <Alert type="info" icon="icon-circle-info">
+            {tct('Check the status of your pending invite requests in the [link].', {
```

Comment:
You can write this as ‘in the [link:Requests tab]’ and then make the link element self closing.

This gives translators better context